.github/workflows/tests.yml

```diff
@@ -7,6 +7,11 @@ on:
       - main
       - 'release/**'
 
+  push:
+    branches:
+      - 'main'
```

Comment:
not quoted on line 7, I guess it doesn't matter

Reply:
It doesn't, but we might as well remove quotes here for consistency.